libs/domain/search/facet-range/facet-range.component.ts

```diff
@@ -122,10 +123,11 @@ export class SearchRangeFacetComponent
     if (!facet) return;
 
     return html`<oryx-search-facet-value-navigation
-      ?open="${this.open}"
-      ?enableClear="${!this.disableClear}"
-      ?dirty="${this.$isDirty()}"
-      .heading="${this.name}"
+      ?open=${this.open}
+      ?enableClear=${!this.disableClear}
+      ?dirty=${this.$isDirty()}
+      .heading=${this.name}
+      .key=${featureVersion >= '1.4' ? facet.parameter : undefined}
```

Comment:
What about using ifDefined directive to make set of attributes a bit cleaner?

`key=${ifDefined(featureVersion >= '1.4' ? facet.parameter : undefined)}`

Reply:
how is it cleaner?